src/privateJrPlatform/getJrPlatformApp.ts

```diff
@@ -23,6 +24,9 @@ async function isValidContentFormat(): Promise<boolean> {
   const siteContentFormat = root.get('contentFormat')
   if (siteContentFormat === CONTENT_FORMAT) return true
 
+  // For alias instances it's better to show a half-broken app, then to show a full error page.
```

Comment:
* [ ] FYI since this behavior changes the documented behavior of the function it would be better to move the comment to the function docs.